net/v2ray-core/Makefile

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright (C) 2019 Xingwang Liao
+# Copyright (C) 2019 Alonzo Liu <unl89@protonmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=v2ray-core
+PKG_VERSION:=4.20.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/v2ray/v2ray-core/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=579a7098b8f6cc68c3f94c5453fa274299f8b072736f364654298bd4cf57d24e
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Alonzo Liu <unl89@protonmail.com>
+
+PKG_CONFIG_DEPENDS := 
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=v2ray.com/core
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-package.mk
+
+define Package/v2ray-core/Default
+  TITLE:=A platform for building proxies
+  URL:=https://www.v2ray.com
+endef
+
+define Package/v2ray-core/Default/description
+Project V is a set of network tools that help you to build your own computer network.
+It secures your network connections and thus protects your privacy.
+endef
+
+define project-v/SubMenu
```

Comment:
Are you planning to add other packages from v2ray as well? If you dont, remove it.
Right now, it doesn't make sense to have your own section for just one package.

![image](https://user-images.githubusercontent.com/4096468/66253090-43fddd00-e763-11e9-95db-4d17db58f414.png)

![image](https://user-images.githubusercontent.com/4096468/66253082-35172a80-e763-11e9-8182-97e9bc079b39.png)